README.md

```diff
@@ -80,6 +80,16 @@ The mapping to CWTN commands is very straight-forward
 | `BurnNow`            | *Not supported*
 | `RaidAssistNum <N>`  | `/rg AssistOutside 1`<br />`/rg OutsideAssistList <Name of Raid MA #N>`
 
+#### rgmercs lua (`rgmercs lua`)
+
+| Boxr command         | Action
+|----------------------|-------------
+| `Camp`               | `/rgl camphard`
+| `Chase`              | `/rgl chaseon`
+| `Manual`             | `/rgl chaseoff`<br/>`/rgl campoff`
+| `BurnNow`            | *Not supported*
```

Comment:
would the command to initiate burn be /varset BurnAlways TRUE so it would be in boxr.cpp file:

void RGMercsLuaControl::BurnNow() {
	boxrRunCommandf("/varset BurnAlways TRUE")

There are no similar burn commands like camp or chase which have /rgl command lines.  I am not sure if this would work to turn burn on if BurnAlways is set to false.

Reply:
In rgl it should be in 

/rgl set show 

Ie

/rgl set BurnAlways 1

But I don't have the code open right now so please double check me